pkgs/amtool-caascad/amtool-caascad.sh

```diff
@@ -0,0 +1,176 @@
+#!/usr/bin/env bash
+
+# Version : AMTOOL_CAASCAD_VERSION
+
+# Usage:
+# amtool-caascad <same options as in amtool> --alertmanager.url=<see below>
+#
+# where --alertmanager.url is one of :
+# - https://<full URL>
+# - <zone>/<subzone> (<zone> from caascad-zones; <subzone> as "caascad", "client", "app" or "consumption")
+#
+
+AMTOOL=amtool
+
+show_help() {
+    cat <<EOHELP
+amtool-wrapper help
+
+This is a wrapper for amtool. Here are the specific options for the wrapper :
+
+  --alertmanager.url : Also accepts "<zone>/<subzone>" format, where <subzone> is one of "caascad", "client", "app" or "consumption".
+                       When a regular URI is specified, it will be used as is.
+
+  --http.config.file : HTTP configuration file as needed by amtool.
+                       However, when it is not specified, and when --alertmanager.url="<zone>/<subzone>", it will
+                       be generated.
+                       When it is explicitely specified, it will override the generated one.
+                       This can be useful when using the same token across multiple calls to Amtool.
+
+  --help             : Show this help before amtool help.
+  --version          : Show the version of the wrapper before amtool version
+
+EOHELP
+}
+
+check_amtool_version() {
+    amtool_version=$(${AMTOOL} --version |grep '^amtool, version' | awk '{print $3}')
+    major=$(echo "${amtool_version}" | cut -d. -f 1)
+    minor=$(echo "${amtool_version}" | cut -d. -f 2)
+    if [ "${major}" -eq "0" ] && [ "${minor}" -lt "24" ]; then
+        echo "amtool version is too old. Found '${amtool_version}'. Need at least version 0.24.0"
+        exit 1
+    fi
+}
```

Comment:
Because you have now amtool from the toolbox in the PATH and you know it's 0.24 you could remove this check if you want

Reply:
Removed all `check_amtool_version` code